Coq-Playground/README.md

```diff
@@ -0,0 +1,61 @@
+# Installation instructions for Coq
+
+The Coq HoTT library requires a bit of a customized installation of Coq.
+You can download pre-built packages for various platforms by following the links here:
+
+  - For Windows: [32 bits]() and [64 bits]()
+  - For [MacOS]()
+  - For Linux distributions: a [snap]() package is available.
+
+These packages come with an Integrated Development Environment, CoqIDE for short,
+which needs to be configured to call a specific Coq version. To do so, simply go 
+to the `Preferences` menu, under `Edit`, then the `External` tab, and set
+`coqidetop` to:
+    
+  On MacOS: `/Applications/Coq_Platform_2021.02.0.app/Contents/Resources/bin/hoqidetop` 
+  On Windows: `???`
```

Comment:
The default installation is in c:\Coq\bin\ I guess

Reply:
Yes on Windows the default is C:\Coq, which can be changed here:

(https://github.com/coq/platform/blob/acddeddad90a9e36a1be5d742c8afaa66960b01d/windows/Coq.nsi#L39)

Reply:
It might make sense to change it to CoqHoTT or so.

Reply:
I think C:\Coq is fine.